### PR TITLE
Limit size for GIF

### PR DESCRIPTION
**Pull request by antho1404**
### Description

Large gifs are creating issues with some crashes on the website on some occasions. Next.js does not optimize gifs, and thus the raw file is loaded and can be extremely resource hungry. 
To limit this issue, I added a limit of 5MB to only GIFs. 5MB is still quite large and will still require quite a lot of bandwidth, but that should be enough quality.

### How to test

- Mint a new token
- Add a gif of more than 5MB
- You should expect an error

@EmmanuelDrouin, if you can have a quick look in terms of UX, that would be great

| no file | file too large |
| --- | --- |
| Screenshot 2023-02-06 at 00 04 25 | Screenshot 2023-02-06 at 00 04 31 |

### CODE REVIEW

**Review by EmmanuelDrouin — COMMENTED** [review]
@antho1404 I tested the system, it works great apart from an issue we are having with the Unlockable content.

If a create an unlockable content with a GIF:
- I upload a GIF as the content to Unlock
- I can create my NFT without issue
- After the creation, the preview crashes 
  - Screen Shot 2566-02-06 at 12 09 35
  - https://nft-test-bsc-testnet-q6pizabgf-liteflow.vercel.app/tokens/97-0xcfbb45ff528c8ff6542887f631a1f704b92cf7db-60249402084937876423066029128237587855293854847399148942248025622398633406578
  - https://nft-test-bsc-testnet-q6pizabgf-liteflow.vercel.app/tokens/97-0xcfbb45ff528c8ff6542887f631a1f704b92cf7db-60249402084937876423066029128237587855293854847399148942288732701670928521081
- On a token detail page I can't interact with the section bellow the media to toggle the preview or hover the tooltip
  - Screen Shot 2566-02-06 at 12 20 02


If I sign out of the platform, then the NFTs that have issue don't have any anymore and are properly displayed (the issue happen as an owner once connected)

**Connected**
Screen Shot 2566-02-06 at 12 21 12

**Not connected**
Screen Shot 2566-02-06 at 12 21 22

**Comment by antho1404** [comment]
I created a dedicated issue for that, I couldn't reproduce it with a gif on my own, maybe related to your gif.
Could you add it on https://github.com/liteflow-labs/starter-kit/issues/162

File: components/Dropzone/Dropzone.tsx [hunk 1]
```diff
@@ -11,7 +11,6 @@ import {
   Text,
```
Comment (NicolasMahe) [inline components/Dropzone/Dropzone.tsx:11]: I can still create an nft with a gif too large in the preview, with a valid with as locked content.

https://user-images.githubusercontent.com/5823445/217221731-8cfa74c2-0422-4453-8626-ec1ccca0c334.mov

**Review by NicolasMahe — APPROVED** [review]
working good 👍

**Pull request merged**